docs/reference/components/sinks/aws_cloudwatch_metrics.cue

```diff
@@ -110,4 +110,17 @@ components: sinks: aws_cloudwatch_metrics: components._aws & {
 			summary:      false
 		}
 	}
+
+	permissions: iam: {
+		platform:  "aws"
+		_service:  "cloudwatch"
+		_docs_tag: "AmazonCloudWatch"
+
+		policies: [
+			{
+				_action:      "PutMetricData"
+				required_for: "healthcheck"
```

Comment:
Is this only for the healthchecks?

Reply:
Hmm, I suppose this can be a list rather than an enum